src/features/søknad/innsending.slice.ts

```diff
@@ -73,25 +73,30 @@ export const sendSøknad = createAsyncThunk<
 
 export interface InnsendingState {
     søknadInnsendingState: RemoteData.RemoteData<ApiError, null>;
+    søknad: RemoteData.RemoteData<ApiError, Søknad>;
```

Comment:
Hvorfor har vi begge disse to? Ser ut som at de alltid blir satt samtidig, så kunne vi ha slått dem sammen?

Reply:
Kunde egentligen kanske haft bara en, skal checka vad den andra bruks till.